tests/test_dagmc_query.py

```diff
@@ -1,68 +1,186 @@
 from pymoab import core, types
 from pymoab.rng import Range
-import dagmc_stats.DagmcFile as ds
+import dagmc_stats.DagmcFile as df
 import dagmc_stats.DagmcQuery as dq
 import pandas as pd
 import numpy as np
 import warnings
 
-test_env = [{'input_file': 'tests/3vols.h5m'}, {
-    'input_file': 'tests/single-cube.h5m'}, {'input_file': 'tests/pyramid.h5m'}]
+test_env = {'three_vols': 'tests/3vols.h5m',
+            'single_cube': 'tests/single-cube.h5m', 'pyramid': 'tests/pyramid.h5m'}
 
 
-def test_get_tris_vol():
+def test_pandas_data_frame():
+    """Tests the initialization of pandas data frames
+    """
+    single_cube = df.DagmcFile(test_env['single_cube'])
+    single_cube_query = dq.DagmcQuery(single_cube)
+    exp_vert_data = pd.DataFrame()
+    assert(single_cube_query._vert_data.equals(exp_vert_data))
+
+    exp_tri_data = pd.DataFrame()
+    assert(single_cube_query._tri_data.equals(exp_tri_data))
+
+    exp_surf_data = pd.DataFrame()
+    assert(single_cube_query._surf_data.equals(exp_surf_data))
+
+    exp_vol_data = pd.DataFrame()
+    assert(single_cube_query._vol_data.equals(exp_vol_data))
+
+def test_get_entities_rootset():
+    """Tests the get_entities function for rootset
+    """
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    three_vols_query = dq.DagmcQuery(three_vols)
+    exp = [three_vols.root_set]
+    assert(three_vols_query.meshset_lst == exp)
+
+def test_get_entities_vol():
+    """Tests the get_entities function for volume meshset
     """
-    Tests the get_tris function for volume meshset
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    vol = three_vols._my_moab_core.get_entities_by_type_and_tag(
+        three_vols.root_set, types.MBENTITYSET, three_vols.dagmc_tags['geom_dim'], [3])[0]
+    three_vols_query = dq.DagmcQuery(three_vols, vol)
+    exp = list(three_vols._my_moab_core.get_child_meshsets(vol))
+    assert(three_vols_query.meshset_lst == exp)
+
+def test_get_entities_surf():
+    """Tests the get_entities function for surface meshset
+    """
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    surf = three_vols._my_moab_core.get_entities_by_type_and_tag(
+        three_vols.root_set, types.MBENTITYSET, three_vols.dagmc_tags['geom_dim'], [2])[0]
+    three_vols_query = dq.DagmcQuery(three_vols, surf)
+    exp = [surf]
+    assert(three_vols_query.meshset_lst == exp)
+
+def test_get_entities_incorrect_dim():
+    """Tests the get_entities function given incorrect dimension
+    """
+    test_pass = np.full(3, False)
+    three_vols = df.DagmcFile(test_env['three_vols'])
+    # check if get_tris function generates warning for meshset with invalid dimension
+    verts = three_vols._my_moab_core.get_entities_by_type_and_tag(
```

Comment:
Consistency/maintainability: in the above tests, you save only the first entity from the range and use it in your query.  Here (and in the next few tests) you save the whole list and then use the first entity in your query.  It is easier for reviewers/future authors to follow if it is consistent (unless it needs to be different, which it doesn't here)